src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -101,9 +101,8 @@
     <Compile Include="Utf8JsonReaderTests.TryGet.Date.cs" />
     <Compile Include="Utf8JsonReaderTests.ValueTextEquals.cs" />
     <Compile Include="Utf8JsonWriterTests.cs" />
-  </ItemGroup>
-  <ItemGroup>
```

Comment:
I would keep these item groups separate (since the other one has a src file being linked in)